src/EventListener/WriteListener.php

```diff
@@ -22,7 +22,7 @@
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Baptiste Meyer <baptiste.meyer@gmail.com>
  */
-class WriteListener
+final class WriteListener
```

Comment:
Can we really do this? It's a BC break, no?

Reply:
Yes! I think we did it in another file recently and we accepted it anyway.

Reply:
We considered it as a bug fix... Because our policy is to make all classes `final` by default. But it's border line I agree.

What I suggest: make it `final`, if someone complain switch to `@final`. It's a brand new class, the chances somebody actually extended it are low.